pandas/core/base.py

```diff
@@ -1185,6 +1185,10 @@ def drop_duplicates(self, keep='first', inplace=False):
               occurrence.
             - False : Mark all duplicates as ``True``.
         take_last : deprecated
+        shortcircuit : boolean, False
+            return True if ANY value is duplicated, False otherwise
```

Comment:
Is this needed? Doesn't give `is_unique` give you the same?

In any case, I am not really a fan of putting it in the same function (keyword results in totally different output type)